docs/admin/service_manager.md

```diff
@@ -43,6 +43,8 @@ multiple requests to be served at a time.
 * `POST /v1/stack/delete/{name}` - Delete the stack removing services and
   networks. Note that volumes, secrets and configs are not deleted.
 
+* `POST /v1/swarm` - Returns the status of managers and workers of the Swarm.
```

Comment:
A post to return status?
How about adding an example of the shape of the returned status?